### DISCUSSION TITLE
Task conversion to issue only available for tasks in first comment of an issue?

### DISCUSSION THREAD

**Discussion by gotom22** [community/community#4503] [Projects and Issues]
It appears that "convert to issue" (button which appears when hovering over a task, i.e. check box item) is only available in the first comment box of an issue. Is this intentional?

- [ ] Test task (edit: feature does not work in discussions)

**Comment by thanhcsf** [comment]
I think it has a reason (I don't know well, I also want to know why Github made this decision). However, I was a bit surprised when you mentioned that it's possible to convert in the first comment, can you show me an example? Because the task display feature in both Issues or the Pull request is only displayed in the detailed description when you create a new Issue or Pull request.

However, I agree with you that counting the total task in an Issue or Pull request (ie the comment section) is a good idea and should be (I remember correctly, Gitlab already has this feature). It's really useful.

**Reply by gotom22** [reply]
hm, I have not found a way to take a screenshot while hovering over the task, so the "convert to issue" pop up shows. 
Any suggestion?

**Reply by thanhcsf** [reply]
You just need to send the link of the public repository with the information you have described. Ex:
```
https://github.com/github/training-kit/issues/811
```

**Reply by gotom22** [reply]
It's not a public repo

**Reply by gotom22** [reply]
https://user-images.githubusercontent.com/776487/124670549-fd25a580-de68-11eb-9eef-30690fab190e.mov

..a bit clumsy, but here's a phone video...
